src/mount.ts

```diff
@@ -28,16 +27,7 @@ interface MountingOptions {
     default?: Slot
     [key: string]: Slot
   }
-  global?: {
-    plugins?: Plugin[]
-    mixins?: ComponentOptions[]
-    mocks?: Record<string, any>
-    stubs?: Record<any, any>
-    provide?: Record<any, any>
-    // TODO how to type `defineComponent`? Using `any` for now.
-    components?: Record<string, Component | object>
-    directives?: Record<string, Directive>
-  }
+  global?: GlobalMountOptions
   stubs?: Record<string, any>
```

Comment:
@lmiller1990 should `stubs` stay here or be moved in `global` ?

Reply:
hmmm good point, probably global yeah? since they are "globally" stubbed.

Reply:
Fixed